lib/charms/prometheus_k8s/v0/prometheus_remote_write.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""This library facilitates the prometheus_remote_write interface.
+
+For the provider charm no communication is necesssary. For the consumer charm,
+instantite a PrometheusRemoteWriteConsumer in the __init__ method of your charm
+class:
+
+```
+from charms.prometheus_k8s.v0.prometheus_remote_write import PrometheusRemoteWriteConsumer
+
+def __init__(self, *args):
+    ...
+    self.remote_write = PrometheusRemoteWriteConsumer(self, "remote-write")
+    ...
+```
+
+Then access the endpoints with:
+`self.remote_write.endpoints`
+or access a prometheus style config object with:
+`self.remote_write.configs`
+"""
+
+from typing import List, Optional, Union
+
+from ops.charm import CharmBase
+from ops.framework import Object
+from ops.model import BlockedStatus
+
+
+class PrometheusRemoteWriteConsumer(Object):
+    """A prometheus remote write consumer."""
+
+    def __init__(self, charm: CharmBase, relation_name: str):
+        """A prometheus remote write consumer.
+
+        Args:
+            charm: The charm object that instantiated this class.
+            relation_name: The relation name as defined in metadata.yaml.
```

Comment:
Shouldn't this be a bit more explicit on which relation name ? For instance "name of the relation for the promethneus_remote_write interface".

Reply:
done